src/pypi_attestations/_impl.py

```diff
@@ -191,6 +191,31 @@ def sign(cls, signer: Signer, dist: Distribution) -> Attestation:
         except ConversionError as e:
             raise AttestationError(str(e))
 
+    @property
+    def claims(self) -> dict[x509.ObjectIdentifier, str]:
+        """Returns the claims present in the Certificate that matches Fulcio OID.
+
+        The complete list is avaible on Fulcio documentation but we only return
+        non deprecated extensions (from 1.3.6.1.4.1.57264.1.8 to .22):
+        https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md
+
+        Values are decoded and returned as strings.
+        """
+        fulcio_oid = x509.ObjectIdentifier("1.3.6.1.4.1.57264.1")
+        certificate = x509.load_der_x509_certificate(self.verification_material.certificate)
+        claims = {}
+        for extension in certificate.extensions:
+            identifier = int(extension.oid.dotted_string.rsplit(".", 1)[1])
+            if (
+                extension.oid.dotted_string.startswith(fulcio_oid.dotted_string)
+                and 8 <= identifier <= 22
+            ):
```

Comment:
Rathen than dynamically building the list of extension OIDs from `.8` to `.22` every time `claims()` is called, I wonder if we should have a constant `SUPPORTED_FULCIO_OIDS: list[x509.ObjectIdentifier]` which lists them all. WDYT?